internal/models/authentication/oauth.go

```diff
@@ -212,8 +212,8 @@ var OAuthProviderAttributes = map[string]schema.Attribute{
 	"client_id":                 stringattr.Optional(),
 	"client_secret":             stringattr.SecretOptional(),
 	"provider_token_management": objectattr.Optional(OAuthProviderTokenManagementAttribute),
-	"prompts":                   listattr.StringOptional(listvalidator.ValueStringsAre(stringvalidator.OneOf("none", "login", "consent", "select_account"))),
-	"scopes":                    listattr.StringOptional(),
+	"prompts":                   strlistattr.Optional(listvalidator.ValueStringsAre(stringvalidator.OneOf("none", "login", "consent", "select_account"))),
```

Comment:
Where are the OAuth grant types, for `implicit` and `authorization code`? We will need these since they are using OneTap with Implicit flow.

This is what they look like from the project snapshot:

```
"allowedGrantTypes": [
        "implicit",
        "authorization_code"
],
```

Also we will need these as well for OAuth:
```
Email and Trust Configuration: `trustProvidedEmails` (indicates whether to trust the email provided by Apple)
Grant and Token Configuration: `revokeUrl` (endpoint to revoke OAuth tokens)
```

Reply:
![image](https://github.com/user-attachments/assets/418984fa-262e-4afa-b716-98f2914d5cd4)
we can, but I think its out of scope for this change, as this is what we support today, worth to oden a new FR

Reply:
@asafshen `allowedGrantTypes` aren't out of scope as I asked for them in the previous issue. I just realized there were there already which I why I resolved the conversation. That's my bad.